### ISSUE TITLE
Does not build on visual studio 2012 even though the documentation says it does.

### ISSUE THREAD

**Issue by fcoulombe** [KhronosGroup/glslang#1570]
It's using variadic templates, initializer lists, alias templates, C99 undefined identifiers, etc
Maybe we should remove the text in the readme saying that it's somewhat supported?

**Comment by johnkslang** [comment]
Did you find those in glslang proper, or in the optional external projects?

**Comment by johnkslang** [comment]
Right. Generally, the external dependencies want to move on to 2015, and less testing will be done with older versions (though 2013 support is still desired). If you wanted to embed glslang source as a parser/translator for GLSL in a project, you would not need the external dependencies. But, they are needed for regression testing and HLSL legalization.  For glslang proper, if you identify build problems with 2012, we can still fix those.
